### DISCUSSION TITLE
Anyone want to help out with writing docs?

### DISCUSSION THREAD

**Discussion by justinclift** [RedashCommunity/redash#10] [General]
Hi all,

Is anyone good at writing docs / blog posts / similar and wants to help out?

We've started adding new features that show up for end users (eg [this PR](https://github.com/RedashCommunity/redash/pull/9)) and having docs about them from near the start would be super useful.

Something like the lines of:

```
2023.05.13 This week in Redash

* @gaecoli enhanced `enum` parameters, so [...]
  [ screenshots ]
  Some description of the change goes here (etc)

* [ other changes ]
```

The same screenshots / text could be assembled and organised into more formal documentation at some point.

Thoughts? :smile:

**Comment by tinder-tder** [comment]
I can try and help out more when i have time, I will try and carve out time next sprint to work on doing what I can here

**Reply by justinclift** [reply]
Cool, thanks. :smile:

My personal situation has changed dramatically a few days ago though, so I can't put time into this any more. :frowning: :frowning_face: :man_facepalming: 

That being said, the wiki here is enabled + you have write access to everything.  So, basic tools are in place...

@SlavchoSpasenoski Any interest in helping out?

**Reply by SlavchoSpasenoski** [reply]
@justinclift 

Sure, I am glad to help where I can do.

Let me know what exactly I can do?

**Reply by justinclift** [reply]
@SlavchoSpasenoski We need to start getting (at least) basic setup docs in place, so people who want to help out can build Redash (from source), so people can create their own docker images, and so on.

And improving / fixing anything broken in the official setup script is probably a good idea too.

On that note, I've sent you an invite to join this new RedashCommunity org. :smile:

With that, you can make changes to things as needed.  eg improve our fork of the [`setup`](https://github.com/RedashCommunity/setup) repo, write/update/etc wiki pages, etc.

Hopefully that makes sense, even though it's a bit open ended... ? :smile:

**Reply by SlavchoSpasenoski** [reply]
@justinclift, Great, I'm In, Thank you for invitation.

**Reply by tinder-tder** [reply]
> Cool, thanks. 😄
> 
> My personal situation has changed dramatically a few days ago though, so I can't put time into this any more. 😦 ☹️ 🤦‍♂️
> 
> That being said, the wiki here is enabled + you have write access to everything. So, basic tools are in place...
> 
> @SlavchoSpasenoski Any interest in helping out?

Sorry to hear, but thanks to taking the initiative to get the ball rolling.  I plan to do some work on this, this week and next pending any fires to put out
